src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -130,4 +134,14 @@ private double getAverageValue(List<UserComment> userCommentList, String scoreMa
                 new RuntimeException(
                     "Unable to calculate average magnitude due to empty input list."));
   }
+
+  private void addToFixedQueue(
+      UserComment newComment, PriorityQueue<UserComment> currentQueue, int queueSize) {
+    if (currentQueue.size() < queueSize) {
+      currentQueue.add(newComment);
+    } else if (newComment.getMagnitude() >= currentQueue.peek().getMagnitude()) {
+      currentQueue.poll();
+      currentQueue.add(newComment);
```

Comment:
May want to add a comment here stating that when two comments have the same magnitude, the one with higher score will be picked, if only could be kept.